### PR TITLE
Channels new PR for OCP 4.3

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: serverless/knative_eventing/serverless-channels.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+The default configuration for channel instances is defined in the `default-ch-webhook` ConfigMap. However, developers can still create their own channels directly by instantiating a supported channel object.
+
+== Supported channel types
+Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
+
+The following are limitations of InMemoryChannel channels:
+
+* No event persistence is available. If a Pod is goes down, events on that Pod will be lost.
+* InMemoryChannel channels do not implement event ordering, so two events that are received in the channel at the same time may be delivered to a subscriber in any order.
+* If a subscriber rejects an event, there are no redelivery attempts. Instead, the rejected event will be sent to a _dead letter sink_ if this exists, or is otherwise dropped.
```
Comment (bobfuru) [inline serverless/knative_eventing/serverless-channels.adoc:22]: "...if this exists, or is otherwise dropped" - i didn't catch this before. It's unclear whether the first part means:
A) "the rejected event is sent to a dead letter sink if **the sink**  exists;
B) "the rejected event is sent to a dead letter sink if **the rejected event**  exists; 

Also unclear whether the second part means:
A) if _xyz_ exists; otherwise, the rejected event is dropped.
B) if _xyz_ exists; otherwise, the sink is dropped.
C) the rejected event will be sent either if _xyz_ exists or if it is dropped.

In other words, I don't get who is doing what to whom in this sentence. :)

**Review by bobfuru — COMMENTED** [review]
A couple of things that I didn't catch earlier in https://github.com/openshift/openshift-docs/pull/21204/

**Comment by bobfuru** [comment]
LGTM - noting that the two edits above can be addressed in a separate PR for master, 4.3, 4.4.

**Comment by abrennan89** [comment]
Adding the two edits to #21322 so they will be in place when this section shows on the site.
Thanks so much @bobfuru for al your help! :confetti_ball:
